gravitee-am-ui/src/app/components/gio-license/gio-license-data.ts

```diff
@@ -13,186 +13,154 @@
+    image: 'assets/gio-license/gravitee-ee-upgrade.svg',
     description:
       'The Twilio resource is part of Gravitee Enterprise. Multi-factor authentication is an additional step during login to enforce access control.',
   },
   // IDPs
-  'am-idp-salesforce': {
-    utm: 'provider_salesforce',
-    image: 'assets/gio-ee-unlock-dialog/am-idp-salesforce.svg',
+  [AmFeature.AM_IDP_SALESFORCE]: {
+    image: 'assets/gio-license/am-idp-salesforce.svg',
     description:
       'The Salesforce identity provider is part of Gravitee Enterprise. Identity providers allow you to configure authentication methods familiar to your users and comply with your security requirement.',
   },
-  'am-idp-saml': {
-    utm: 'provider_saml',
-    image: 'assets/gio-ee-unlock-dialog/am-idp-saml.svg',
+  [AmFeature.AM_IDP_SAML]: {
+    image: 'assets/gio-license/am-idp-saml.svg',
     description:
       'The SAML 2.0 identity provider is part of Gravitee Enterprise. Identity providers allow you to configure authentication methods familiar to your users and comply with your security requirement.',
   },
-  'am-idp-ldap': {
-    utm: 'provider_ldap',
-    image: 'assets/gio-ee-unlock-dialog/am-idp-ldap.svg',
+  [AmFeature.AM_IDP_LDAP]: {
+    image: 'assets/gio-license/am-idp-ldap.svg',
     description:
       'The LDAP identity provider is part of Gravitee Enterprise. Identity providers allow you to configure authentication methods familiar to your users and comply with your security requirement.',
   },
-  'am-idp-kerberos': {
-    utm: 'provider_kerberos',
-    image: 'assets/gio-ee-unlock-dialog/am-idp-kerberos.svg',
+  [AmFeature.AM_IDP_KERBEROS]: {
+    image: 'assets/gio-license/am-idp-kerberos.svg',
     description:
       'The Kerberos identity provider is part of Gravitee Enterprise. Identity providers allow you to configure authentication methods familiar to your users and comply with your security requirement.',
   },
-  'am-idp-azure-ad': {
-    utm: 'provider_azure_ad',
-    image: 'assets/gio-ee-unlock-dialog/am-idp-azure-ad.svg',
+  [AmFeature.AM_IDP_AZURE_AD]: {
+    image: 'assets/gio-license/am-idp-azure-ad.svg',
     description:
       'The Azure AD identity provider is part of Gravitee Enterprise. Identity providers allow you to configure authentication methods familiar to your users and comply with your security requirement.',
   },
-  'am-idp-france-connect': {
-    utm: 'provider_france_connect',
-    image: 'assets/gio-ee-unlock-dialog/am-idp-france-connect.svg',
+  [AmFeature.AM_IDP_FRANCE_CONNECT]: {
+    image: 'assets/gio-license/am-idp-france-connect.svg',
     description:
       'The France Connect identity provider is part of Gravitee Enterprise. Identity providers allow you to configure authentication methods familiar to your users and comply with your security requirement.',
   },
-  'am-idp-cas': {
+  [AmFeature.AM_IDP_CAS]: {
     // utm_source=oss_am&utm_medium=provider_cas&utm_campaign=oss_am_to_ee_am
```

Comment:
Not that all `utm`s are gone here, maybe this comment can be also removed

Reply:
✅